src/main/resources/bundleconfiguration/iac-upper-tribunal-bundle-config.yaml

```diff
@@ -0,0 +1,72 @@
+title: IAC Upper Tribunal Bundle
+filename: Upper-Tribunal-Bundle.pdf
+filenameIdentifier: /case_details/case_data/bundleFileNamePrefix
+coverpageTemplate: TB-IAC-GNO-ENG-00746.docx
+hasTableOfContents: true
+hasCoversheets: true
+documentNameValue: /document/document_filename
+documentLinkValue: /document
+sort:
+  field: /dateUploaded
+  order: ascending
+hasFolderCoversheets: true
+pageNumberFormat: numberOfPages
+folders:
+  - name: Appellant FTPA documents
+    documents:
+      - type: documentSet
+        property: /ftpaAppellantDocuments
+  - name: Respondent FTPA documents
+    documents:
+      - type: documentSet
+        property: /ftpaRespondentDocuments
+  - name: Decisions and reasons documents
+    documents:
+      - type: documentSet
+        property: /finalDecisionAndReasonsDocuments
+  - name: Hearing documents
+    documents:
+      - type: documentSet
+        property: /hearingDocuments
+        filters:
+          - property : /tag
+            value : caseSummary
+      - type: documentSet
+        property: /hearingDocuments
+        filters:
+          - property: /tag
+            value: hearingRequirements
+      - type: documentSet
+        property: /hearingDocuments
+        filters:
+          - property: /tag
+            value: hearingNotice
+  - name: Legal representative documents
```

Comment:
that will become "Appellant documents" soon, worth to confirm with Frank what we want to see in bundle

Reply:
https://tools.hmcts.net/jira/browse/RIA-4510

Reply:
Correct. Frank has confirmed this collection name change to "Appellant documents" (as per his comment in AIP ticket RIA-4510). He's added similar comment to ticket RIA-4351.